prowler/providers/aws/services/s3/s3_service.py

```diff
@@ -285,9 +312,14 @@ def __get_bucket_tagging__(self, bucket):
         except ClientError as error:
             bucket.tags = []
             if error.response["Error"]["Code"] != "NoSuchTagSet":
-                logger.error(
-                    f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
-                )
+                if error.response["Error"]["Code"] == "NoSuchBucket":
```

Comment:
should it be at the same level?

Reply:
Nope, we don't want to log the exception when the error is `NoSuchTagSet`

Reply:
Good one, thanks for the explanation!